includes/class-activitypub.php

```diff
@@ -703,6 +704,36 @@ public static function prevent_empty_post_meta( $check, $object_id, $meta_key, $
 		return $check;
 	}
 
+	/**
+	 * Get post meta.
```

Comment:
```suggestion
	 * Adjusts default post meta values.
```
Maybe something like that? Could probably still be more descriptive